routing/pathfind.go

```diff
@@ -368,12 +366,9 @@ func findPath(g *graphParams, r *RestrictParams, source, target route.Vertex,
 
 	// processEdge is a helper closure that will be used to make sure edges
 	// satisfy our specific requirements.
-	processEdge := func(fromNode *channeldb.LightningNode,
-		edge *channeldb.ChannelEdgePolicy,
+	processEdge := func(fromVertex route.Vertex, edge *channeldb.ChannelEdgePolicy,
```

Comment:
nit: line a tiny bit too long?

Reply:
Fixed this by switching arguments to make the lines be under <= 80 chars